proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md

```diff
@@ -0,0 +1,78 @@
+# Clarification of SDL App Icon display sequence and display order
+
+* Proposal: [SDL-NNNN](NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Policy Server ]
+
+## Introduction
+
+This proposal is to clarify the SDL App Icon display sequence and the display order of icons. By doing so, we can prevent unnecessary development for the HU developers and improve usabilty by enhancing the visuality for the users.
+
+## Motivation
+
+Currently, the SDL app icon is displayed on the SDL app selection (menu) screen, but the sequence to display the icon is not provided. Also, there is no explicit definition of the icon's display order and the rule for example grouping by app type, as a result, the display order of icons changes every launch. Incidentally, it is implemented on the program code of SDL Core by the following rule.
+
+ 1. Registered apps are sorted by the ascending order of appID *1.
+ 2. Unregistered apps are sorted by the ascending order of policyID in the policy table. If there are same policyIDs, they will be sorted by the ascending order of deviceID.
+
+*1 appID: It is NOT the ID obtained during App Certification. It is the ID that is set when the app establish session and be launched based on the hash algorithm using the sessionID (set from 0 in the app registration order) and deviceHandlerID (set from 0 in the device connection order).
+
+For these reasons, it is difficult for the HU developer to know what specification the icons are sorted. Also, it is difficult to judge whether or not it is necessary to deal with it on the HMI side. Furthermore, it is very poor visualilty and operability (usability) because it takes a long time for the user to find an app that s/he wants to use. In order to solve these problems, it clarify the display order of the SDL App Icon and improve the usability by presenting the sequence and the rules of the SDL App Icon display.
```

Comment:
```suggestion
For these reasons, it is difficult for the HU developer to know by what specification the icons are sorted. Also, it is difficult to judge whether or not it is necessary to deal with it on the HMI side. Furthermore, it is very poor visuality and operability (usability) because it takes a long time for the user to find an app that s/he wants to use. In order to solve these problems, we propose to clarify the display order of the SDL app icon and improve the usability by presenting the sequence and the rules of the SDL app icon display.
```